dataland-backend/build.gradle.kts

```diff
@@ -55,13 +53,13 @@ dependencies {
     testImplementation(Testing.mockito.core)
     testImplementation(Spring.security.spring_security_test)
     implementation(project(":dataland-keycloak-adapter"))
-    implementation(libs.mailjet.client)
     implementation(project(":dataland-message-queue-utils"))
     implementation(libs.flyway)
     implementation(libs.flyway.core)
     implementation("org.springframework.boot:spring-boot-starter-validation:3.1.5")
     implementation("org.springframework.boot:spring-boot-starter-web")
     implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+    implementation(libs.json)
```

Comment:
Wieso ist das notwendig geworden?

`org.json` Sachen werden auch in main im backend genutzt, ohne dass diese dependency hier aufgeführt wird.

Reply:
Ich weiß nicht mehr konkret was es war. Aber wir mussten das einbinden um irgendeine dependency zu satisfyen, die wahrscheinlich auch von irgendeiner anderen mitgeliefert wird im backend etc. Kann ich nochmal ausprobieren.

Reply:
Achso. Das ist ja das backend file. Im Backend verwenden wir org.json.JSONObject. Das kam wahrscheinlich vorher aus mailjet irgendwie mit 🤷 Im email service kann ich darauf zugreifen ohne weitere dependency. Das backend kann deswegen wie gesagt nicht bauen, wenn ich die dependency rauswerfe.

Reply:
Ah ok ja das kann sein.

Dann passts.